_content/news/_posts/2019-01-14-hervorming.markdown

```diff
@@ -0,0 +1,24 @@
+---
+layout: post
+title: "Hervorming jeugdcompetitie"
+date: 2019-01-07 10:00:00
+description: Volgend seizoen gaat de jeugdcompetitie er heel anders uitzien
+cover: /news/img/2018-12-28-lizards-grid-rsz.jpg
+permalink: /news/2019-01-14-hervorming/
+---
+
+Op de Algemene Vergadering van basketbal vlaanderen werd in november 2018 de hervorming van de jeugdcompetitie goedgekeurd.
+
+Doel is een evenwichtigere competitie bij de jeugd en een goed gevulde kalender, zodat jonge spelers optimale leerkansen krijgen op het terrein. Dit zowel op het hoogste niveau, als alle niveaus eronder. Verder wil de bond op deze manier de clubs ook aanzetten tot een betere jeugdwerking en actievere recrutering.
```

Comment:
Doel is een evenwichtigere competitie bij de jeugd en een goed gevulde kalender op alle niveaus, zodat jonge spelers optimale leerkansen krijgen op het terrein. Verder wil de VBL op deze manier de clubs ook aanzetten tot een betere jeugdwerking en actievere recrutering.